alws/release_planner.py

```diff
@@ -395,7 +395,9 @@ async def commit_release(
 
         builds_released = False
         try:
-            release_messages = await self.execute_release_plan(release)
+            release_messages, release_plan = await self.execute_release_plan(
+                release,
+            )
```

Comment:
Can we just make `message` variable here like `'\n'.join(release_messages)` and skip all other changes? I just do not understand a point of returning release plan from the execution function

Reply:
Good point, I'll check if we modifying release plan or not during `execute_release_plan`

Reply:
As I thought, we modify release plan during execution
https://github.com/AlmaLinux/albs-web-server/blob/9fb851dc25611fa2c3a7c98e3464f7ced31055df/alws/release_planner.py#L1245-L1252
We can send copy of release plan into `execute_release_plan` like it was before or just use current approach